odl/operator/default_ops.py

```diff
@@ -579,6 +585,211 @@ def __str__(self):
         return "{}.T".format(self.vector)
 
 
+class NormOperator(Operator):
+    """Operator taking the norm of a vector.
+
+    ``NormOperator(space)(x) <==> space.norm(x)``
+
+    This is only applicable in normed spaces.
+
+    See Also
+    --------
+    InnerProductOperator : Inner product with fixed vector.
+    DistOperator : Distance to fixed vector.
+    """
+
+    def __init__(self, space):
+        """Initialize a NormOperator instance.
+
+        Parameters
+        ----------
+        space : `LinearSpace`
+            Space to take the norm in.
+        """
+        super().__init__(space, RealNumbers(), linear=False)
+
+    def _call(self, x):
+        """Return the norm of x.
```

Comment:
```
``x``
```
We have a bit mixed style for the `_call` docstring. Sometimes we say
```
Return ``self(x)``.
```
or implement in the case of `out`, but sometimes we have some other text. Would be good to have some common style.